recipes-bsp/u-boot/files/mx8m-generic-bsp/u-boot-hab.cfg

```diff
@@ -2,3 +2,4 @@ CONFIG_IMX_HAB=y
 CONFIG_SPL_DRIVERS_MISC=y
 CONFIG_SPL_CRYPTO=y
 CONFIG_ARCH_MISC_INIT=y
+CONFIG_SPL_LOAD_FIT_ADDRESS=0x44000000
```

Comment:
The commit message for this says this is a "short term solution". Out of curiosity do you already have in mind a "long-term" solution for this?

Reply:
Yes, depending on the result of the conversation with the BSP team (mentioned in the description) we should hopefully be able to move this setting to the defconfigs of verdins (or to a BSP layer). But the solution is the same, what changes is only where we'd apply it. Applying it on this security layer is arguably not ideal because the layer shouldn't be concerned with the location of artifacts unrelated to secboot on the device memory space.

Reply:
I started the conversation and today I noticed they created a ticket to handle this on their side and it seems this will be given priority. The change will most likely go straight into the upstream U-Boot defconfig for the verdin-imx8m/p. I imagine it will take a couple of weeks until all work is done. I'm following up their ticket so that when it's done I'd make the change to the present layer as well.

Reply:
Sounds good, thanks!